client/src/constants/apiConstants.js

```diff
@@ -0,0 +1,21 @@
+const API_SOURCE_NAME = 'name';
+const API_SOURCE_TYPE = 'source_type';
+const API_SOURCE_HOSTS = 'hosts';
+const API_SOURCE_PORT = 'port';
+const API_SOURCE_CREDENTIALS = 'credentials';
+const API_SOURCE_SAT_VERSION = 'satellite_version';
+const API_SOURCE_SSL_CERT = 'ssl_cert_verify';
+const API_SOURCE_SSL_PROT = 'ssl_protocol';
+const API_SOURCE_SSL_DISABLE = 'disable_ssl';
```

Comment:
Hmmm.... Guess this should be done everywhere then. Could get quite lengthy.

Reply:
Yeahhhhhh, was debating on it... I figured it would be nice not to have to worry about field naming changes. Had planned on trying to limit its usage around submitting the data. Originally had it incorporated all over the place, ughhhhh that was awful.

Reply:
OK, we'll take it a step at a time.